test/Microsoft.Azure.WebJobs.Host.UnitTests/Configuration/WebJobsExtensionOptionsConfigurationTests.cs

```diff
@@ -0,0 +1,70 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.Azure.WebJobs.Host.Configuration;
+using Microsoft.Extensions.Configuration;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Xunit;
+
+namespace Microsoft.Azure.WebJobs.Host.UnitTests.Configuration
+{
+    public class WebJobsExtensionOptionsConfigurationTests
+    {
+        [Fact]
+        public void Configuration_IsBound()
+        {
+            var config = new Dictionary<string, string>
+            {
+                { "AzureWebJobs:extensions:testext:testvalue1", "test1" },
+                { "azurewebjobs:extensions:testext:testvalue2", "test2" },
+            };
+
+            TestOptions options = ConfigureOptions<TestOptions>(config, "testext");
+
+            // Assert
+            Assert.Equal("test1", options.TestValue1);
+            Assert.Equal("test2", options.TestValue2);
+        }
+
+        [Fact]
+        public void Configuration_WithConfiguredRootSectionName_IsBound()
+        {
+            var config = new Dictionary<string, string>
+            {
+                { "AzureWebJobsConfigurationSection", "testroot" },
```

Comment:
This is what Functions will do to swap this out, right?

Reply:
Yes. This makes the root configurable.